src/ja/user/main.py

```diff
@@ -1,4 +1,35 @@
+from typing import List
+
+from ja.user.proxy import UserServerProxy
+from ja.user.cli import UserClientCLIHandler
+from ja.user.message.add import AddCommand
+from ja.user.message.cancel import CancelCommand
+from ja.user.message.query import QueryCommand
+
+
 class JobAdder:
     """
     The main class for the JobAdder user client.
     """
+    def __init__(self, config_path: str = "~/.config/jobadder", remote_module: str = "ja.server.proxy.remote",
+                 command_string: str = "python3 -m %s") -> None:
+        self._cli_handler = UserClientCLIHandler(config_path=config_path)
+        self._remote_module = remote_module
+        self._command_string = command_string
+
+    def run(self, cli_args: List[str] = None) -> None:
```

Comment:
what is the purpose of the `cli_args`?

Reply:
I plan to use this for the integration tests.